README.md

```diff
@@ -48,26 +48,52 @@ CLX is available in a Docker container, by building from source, and through Con
 
 ### CLX Docker Container
 
-Pull the RAPIDS container and build for CLX.
+Prerequisites
+* NVIDIA Pascal™ GPU architecture or better
+* CUDA 9.2 or 10.0 compatible NVIDIA driver
+* Ubuntu 16.04/18.04 or CentOS 7
+* Docker CE v18+
+* nvidia-docker v2+
+
+Pull the RAPIDS container and build CLX image.
 
 ```aidl
 docker pull rapidsai/rapidsai-dev-nightly:0.11-cuda10.0-devel-ubuntu18.04-py3.7
-docker build -t clx .
+docker build -t clx:latest .
+```
+
+Start Container and Notebook Server
+
+#### Preferred - Docker CE v19+ and nvidia-container-toolkit
+```aidl
+docker run  --gpus '"device=0"' \
+  --rm -d \
+  -p 8888:8888 \
+  -p 8787:8787 \
+  -p 8686:8686 \
+  clx:latest
+```
+
+#### Legacy - Docker CE v18 and nvidia-docker2
+```aidl
 docker run --runtime=nvidia \
```

Comment:
I guess technically if you want to match the `--gpus 'device=0'` above, you should put `-e NVIDIA_VISIBLE_DEVICES=0` here.